base/precompilation.jl

```diff
@@ -338,6 +338,9 @@ end
 const Config = Pair{Cmd, Base.CacheFlags}
 const PkgConfig = Tuple{Base.PkgId,Config}
 
+# TODO: Remove once Pkg moves to the renamed kwarg `flags_cacheflags` -> `configs`
+precompilepkgs(args...; flags_cacheflags, kwargs...) = precompilepkgs(args...; configs=flags_cacheflags, kwargs...)
```

Comment:
I don't think this works since the positional arguments are the same? Probably need to handle it in the body of the original function?

Reply:
Ah ok. Fixed